gammapy/maps/core.py

```diff
@@ -132,6 +132,24 @@ def quantity(self, val):
         self.data = val.value
         self._unit = val.unit
 
+    def rename_axes(self, names, new_names):
+        """Rename the Map axes.
+    
```

Comment:
Maybe specify that this will create a full copy of the `Map` as this might not be obvious from the name.

Reply:
By default all maps methods do a copy, except for `.stack()`,  so I think it's fine to just state the exception.